tests/scripts/test-e2e-standalone.sh

```diff
@@ -92,4 +92,4 @@ if [ $STARTED -ne 1 ]; then
 fi
 
 echo Done waiting for OLS server start, running e2e
-make test-e2e
+SUITE_ID=standalone TEST_TAGS="not cluster" make test-e2e
```

Comment:
Not a comment about this particular line, just can't anchor it anywhere else:
We can replace the the OLS readiness code above with wait_for_ols() from utils.sh

Reply:
updated